docs/external plugins/Verifier/sbom.md

```diff
@@ -0,0 +1,294 @@
+# SBOM Validation
+
+This document outlines how Ratify can be used to verify sbom (Software bill of material). The `sbom` verifier is added as a plugin to the Ratify verification framework.  Currently the sbom verifier is in 2.0.0-alpha release, and it supports the following sbom validation:
+- sbom attached to the subject image as a referrer artifact
+- sbom generated in spdx+json format
+
+## Table of Contents
+
+* [SBOM with License and Package Validation](#sbom-with-license-and-package-validation)
+* [SBOM with Signature Validation](#sbom-with-signature-validation)
+* [Configuration](#configuration)
+* [Future improvement](#future-improvements)
+* [FAQ](#faq)
+
+## SBOM with License and Package Validation
+
+Alice has a Kubernetes cluster. The software she deploys to her cluster depends on many open source components. She wants to make sure the container images meet the following criteria:
+- does not contain licenses that could conflict with her business interest
+- does not contain any vulnerable packages
+
+### Walkthrough
+
+#### 1. Generate SBOM and attach to your image
+
+Use a sbom generator such as syft to generate an sbom for your iamge  `myregistry.io/sbom/alpine:3.18.2`. A reference artifact is generated:
+1. Use syft to scan `myregistry.io/sbom/alpine:3.18.2` and save the output file
+    ```shell
+    syft -o spdx-json --file sbom.spdx.json myregistry.io/sbom/alpine:3.18.2
+    ```
+    
+2. A tool such as `oras` is used to package, attach, and then push the report to registry
+    - `artifact-type` MUST be `application/spdx+json`
+    ```shell
+    oras attach \
+        --artifact-type application/spdx+json \
+        myregistry.io/sbom/alpine:3.18.2 \
+        sbom.spdx.json:application/spdx+json
+    ```
+
+The resulting image will have a single SBOM artifact attached:
+
+```shell
+> oras discover myregistry.io/sbom/alpine:3.18.2 -o tree
+myregistry.io/sbom/alpine@sha256:96f270a2d97f70713ef7bd6c4b80552178bf97fc6bd75ac9af2df4ba06b26f62
+└── application/spdx+json
+    └── sha256:6944ae19f248ed93a494c528a839d3eac4c33df6ca81d6f762a0483af8b2b87f
+```
+
+#### 2. Ratfy Installation and configuration
+First, follow the first step of the [manual quickstart](../../quickstarts/quickstart-manual.md) to installs Gatekeeper on the cluster. 
+
+Second, install Ratify and configure the sbom verifier with disallowed license and package information. In the configuration below, Alice specifies `busybox` as a disallowed package as it leads to arbitrary code execution. Copy left license `MPL` is also disallowed due to license restrictions.
```

Comment:
I think we're missing the step to install the constraint and constraint template?